optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/vehicleRoutingSolverConfig.xml

```diff
@@ -80,4 +80,44 @@
       <acceptedCountLimit>1</acceptedCountLimit>
     </forager>
   </localSearch>
+
+  <ruin>
+    <ruinMoveSelector>
```

Comment:
This nesting appears unnecessary. There will never be any selector here, other than a ruin selector, right?

I'd rather see the config without the `ruinMoveSelector`, going from `<ruin>` straight to `<percentage...>`, `<entitySelector>` etc.